shared/version/api.go

```diff
@@ -245,6 +245,7 @@ var APIExtensions = []string{
 	"projects_limits_instances",
 	"network_state_vlan",
 	"instance_nic_bridged_port_isolation",
+	"gvrp",
```

Comment:
@TimRots this should be `network_gvrp` to match the name of the extension in the docs.

Reply:
Thanks, resolved this one.